Assets/Scripts/Augment/BulletModifiers/StickyProjectileModifier.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// A modifier to stick a gameObject (stuckObject) to a target after hitting said target.
+/// </summary>
+public class StickyProjectileModifier : ProjectileModifier
+{
+    // Model to stick to target
+    [SerializeField]
+    private GameObject stuckObject;
+    [SerializeField]
+    private float stuckLifeTime = 5f;
+    // Does stuck object trigger onHit?
+    [SerializeField]
+    private bool triggerOnHit = true;
+    // How often does it trigger onHit (in seconds)?
+    [SerializeField]
+    private float onHitInterval = 1f;
+
+    public void StickToTarget(HitboxController controller, ref ProjectileState state, GunStats stats)
+    {
+        var stuck = Instantiate(stuckObject, state.position, state.rotation, controller.transform);
```

Comment:
Perhaps adjust the position before instancing? Sometimes you only see the brim of the hat:

![image](https://user-images.githubusercontent.com/44986729/220334798-4f9db547-48d1-4080-8d54-15759214d36d.png)

Reply:
Hope this satisfies all hat-pincushion-related needs 🤠
![image](https://user-images.githubusercontent.com/54811121/220433578-bd1d0e87-c6c8-451c-92c5-f8d5b333ac72.png)